src/components/Discover/index.tsx

```diff
@@ -44,6 +46,15 @@ const Discover = () => {
       }
     );
 
+  const { data: watchlistItems, error: watchlistError } = useSWR<{
+    page: number;
+    totalPages: number;
+    totalResults: number;
+    results: WatchlistItem[];
+  }>('/api/v1/discover/watchlist', {
+    revalidateOnMount: true,
+  });
```

Comment:
We can check if the user is a Plex user and only make this API call if they are.